packages/extension/src/newtab/App.tsx

```diff
@@ -104,6 +123,18 @@ function InternalApp({
       : DEFAULT_TAB_TITLE;
   }, [unreadCount]);
 
+  useEffect(() => {
+    if (shouldRedirectOnboarding) {
+      routeChangedCallbackRef.current();
+    }
+  }, [routeChangedCallbackRef, shouldRedirectOnboarding]);
+
+  if (shouldRedirectOnboarding) {
+    // eslint-disable-next-line no-param-reassign
+    pageRef.current = '/hijacking';
```

Comment:
`/onboarding`? Or does `hijacking` has some other meaning for analytics?

Reply:
Its a analytic split between onboarding page and this extension only one.
Didnt have a better name 😂